core/schemas/__init__.py

```diff
@@ -0,0 +1,25 @@
+import importlib
+import inspect
+from pathlib import Path
+
+import aenum
```

Comment:
Is this a built in or new dependency?

Reply:
Not a built in dependency. I'm still thinking about another way dynamically build the Enum maybe by implementing `_missing_` method. This could prevent the use of an external dependency.

Reply:
It's fine if it's an external dep, but I just don't see the updates to the .toml and .lock files :)

Reply:
fixed!